pkg/codegen/docs.go

```diff
@@ -16,9 +16,11 @@ package codegen
 
 import (
 	"regexp"
-	"strings"
+
+	"github.com/pgavlin/goldmark/ast"
 
 	"github.com/pulumi/pulumi/pkg/v2/codegen/schema"
+	"github.com/pulumi/pulumi/sdk/v2/go/common/util/contract"
 )
 
 var (
```

Comment:
Can we now get rid of the regex's from these [lines](https://github.com/pulumi/pulumi/pull/4838/files#diff-1bed47e82076977fc5ffa09e15aa568dL25-L51)?

Reply:
As long as they're not used by any consuming packages, yes. I wasn't sure if e.g. the docs site used them.

Reply:
Yeah since the functions that were using them have been removed with this PR, we should be fine to remove them assuming the build also passes.